project/Dependencies.scala

```diff
@@ -39,7 +37,7 @@ object Dependencies {
     "com.gu" %% "fastly-api-client" % "0.2.6",
     "com.fasterxml.jackson.dataformat" % "jackson-dataformat-yaml" % Versions.jackson,
     "com.fasterxml.jackson.core" % "jackson-databind" % Versions.jackson,
-    "com.typesafe.play" %% "play-json" % Versions.play
+    "com.typesafe.play" %% "play-json" % "2.6.13"
```

Comment:
`Play` and `play-json` versions are no more aligned and have separate lifecycles.